archiver.go

```diff
@@ -107,6 +121,7 @@ func (a *Archiver) do(log log15.Logger, j *Job) (err error) {
 			finalErr = ErrClone.Wrap(err, endpoint)
 		}
 
+		r.Status = model.Pending
```

Comment:
Or we could simply add a `model.FetchStatus` parameter to `dbUpdateFailedRepository` and make it explicit that it needs to receive a status